docs/repairs-api/sor-engine.md

```diff
@@ -38,205 +38,108 @@ Not all journey's have 3 tier's, e.g. Kitchen - Worktop
 :::
 
 ### Configuration
-The SoR engine is configured by specifying the SoR codes in [SoRConfig.json](https://github.com/City-of-Lincoln-Council/housing-repairs-online-api/blob/main/HousingRepairsOnlineApi/SoRConfig.json).
+The SoR engine is configured by specifying the `SOR_CONFIGURATION` environment variable.
 
-Each existing SoR code must exist and have a value.
+The configuration allows specifying the key value, display text and either options (used to gather additional information) or a SoR code.
 
-If an existing SoR code is removed, the mapping resolution will fail.
-If a new SoR code is added, it will have no effect. (The frontend would need to be updated to offer the new selection options.)
+The `SOR_CONFIGURATION` value is used to determine which options to offer at each tier (where, what and best description) and a SoR code for the combination of options.  
```

Comment:
`the best description`? (I'm not sure, I'm not a native speaker, @CatherineDeas would know best probably)

Reply:
here no need to put "the" as it's "telegram" style